gulpfile.js

```diff
@@ -38,7 +38,8 @@ const watchedSources = [
 ];
 
 const scripts = [
-    'src/terminateProcess.sh'
+    'src/terminateProcess.sh',
+    'src/launchUnelevated.js'
```

Comment:
I think this will have to be added to the csproj for signing

Reply:
Looking at the current state of the csproj, when it uses wildcard of `..\out\src\**\*` and `$(OutDir)\**\*.js`, it should include this file too.